.github/workflows/release.yml

```
@@ -22,23 +22,23 @@
       contents: write  # Required for creating releases
       id-token: write # Required for signing
       packages: write # Required for container publishing
-    
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)